lockc/src/runc.rs

```diff
@@ -100,9 +103,9 @@ fn container_type_data<P: AsRef<std::path::Path>>(
     // Kubernetes
     if let Some(annotations) = config.annotations {
         debug!(
-            "detected kubernetes container with bundle {}, config {}",
-            bundle_path.display(),
-            config_path.display(),
+            bundle = bundle_path.display().to_string().as_str(),
+            config = config_path.display().to_string().as_str(),
```

Comment:
tracing handles that in a nice way, there's no need to call `display()...`:

```rust
debug!(
  bundle = %bundle_path,
  config = ?config_path,
  "bundle uses display, while config uses debug");
```

This is documented [here](https://docs.rs/tracing/latest/tracing/index.html#recording-fields)

Reply:
Doesn't work for me

```
error[E0277]: `std::path::Path` doesn't implement `std::fmt::Display`
   --> lockc/src/runc.rs:106:23
    |
106 |             bundle = %bundle_path,
    |                       ^^^^^^^^^^^ `std::path::Path` cannot be formatted with the default formatter; call `.display()` on it
    |
    = help: the trait `std::fmt::Display` is not implemented for `std::path::Path`
    = note: call `.display()` or `.to_string_lossy()` to safely print paths, as they may contain non-Unicode data
    = note: this error originates in the macro `format_args` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Reply:
fixed